### DISCUSSION TITLE
Unable to see website on Github Pages

### DISCUSSION THREAD

**Discussion by ishan-soni27** [community/community#133443] [Programming Help] [answered]
### Body

Hello, I am trying to create a website for my friend and tried to set up a github pages website so I could test it. The website is active and running, however, it is only rendering the navbar and not my other pages. Furthermore, when I click on the options on the navbar, it takes me to an error page. I am not that experienced with React, so I am a bit clueless as to why it wont render the pages. Is it because of the way I set up the pages with .jsx files? I will attach the site and the repo here as well. The website works fine when I run it on my computer and renders everything correctly. 

Repo: https://github.com/Fentrai/CadburryOfficialWebsite
GitHub Pages Link: https://fentrai.github.io/CadburryOfficialWebsite/

Thank you very much, your help is highly appreciated. 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by LuaanNguyen** [comment]
Hi @Fentrai ,

Try deploying it on Vercel instead. All you need to do is link your Github to Vercel. Then all your repo can be deployed on Vercel dashboard. This is more useful later in your development. 

You can find all the information at : [Vercel Project Deployment](https://vercel.com/docs/getting-started-with-vercel/projects-deployments )

If you have any questions. Let me know.

**Reply by ishan-soni27** [reply]
Thanks, I tried it out, however, the screen is just blank here and nothing is showing up. It's just blank. Do you think there could be an internal error? Is it because I used React?

Link: https://cadburry-official-website.vercel.app/

**Reply by LuaanNguyen** [reply]
@Fentrai I've been trying to deploy it on Github Pages as well as Vercel. I forked your repo and did a little changes here and there (you can see my commit history). It is now working and deployed on my own version: https://cadburry-official-website-kappa.vercel.app/ 

All you need to do now is accept my pull request so it syncs my version with yours and you should be good to go.

**Reply by ishan-soni27** [reply]
Thank you so much man! I really really appreciate this. I can't thank you enough. Is there any way I could pay you or something?

**Reply by LuaanNguyen** [reply, marked as answer]
No problem man. I recommend you use Vercel to deploy any of your front-end project in the future. You don't have to pay anything. Have a nice day!
